packages/synapse-react-client/src/components/SynapseChat/SynapseChat.tsx

```diff
@@ -279,46 +302,40 @@ export const SynapseChat: React.FunctionComponent<SynapseChatProps> = ({
                 >
                   <SynapseSpinner size={40} />
                   {/* Show the current message, as well as the full trace log in a tooltip */}
-                  {traceEvents && traceEvents.page && (
-                    <Box sx={{ position: 'relative', pt: '35px' }}>
-                      <TransitionGroup>
-                        {/* The key is based on the current text so when the text changes, the Fade component will remount */}
-                        <Fade key={traceEvents.page[0].message} timeout={500}>
-                          <Tooltip
-                            placement="bottom"
-                            title={
-                              <div style={{ textAlign: 'center' }}>
-                                {traceEvents?.page
-                                  ?.slice()
-                                  .reverse()
-                                  .map((event, index) => {
-                                    return (
-                                      <Typography
-                                        key={`${index}-${event.message}`}
-                                      >
-                                        {event.message}
-                                      </Typography>
-                                    )
-                                  })}
-                              </div>
-                            }
+                  <Box sx={{ position: 'relative', pt: '35px' }}>
+                    <TransitionGroup>
+                      {/* The key is based on the current text so when the text changes, the Fade component will remount */}
+                      <Fade key={latestTraceEventMessage} timeout={500}>
+                        <Tooltip
+                          placement="bottom"
+                          title={
+                            <div style={{ textAlign: 'center' }}>
+                              <Typography>Initializing...</Typography>
```

Comment:
Probably only show this node if `allTraceEvents` is empty

Reply:
nice catch